test/test_linalg.py

```diff
@@ -410,6 +410,103 @@ def run_test_case(input, p, dim, keepdim):
                 for ord in ord_settings:
                     run_test_case(input, ord, dim, keepdim)
 
+    @skipCPUIfNoLapack
+    @skipCUDAIfNoMagma
+    @dtypes(torch.float32, torch.float64, torch.complex64, torch.complex128)
+    @dtypesIfCUDA(torch.float32, torch.float64)
+    @precisionOverride({torch.float32: 1e-3})
```

Comment:
Is this because NumPy computes the condition number in double?

Reply:
Yes, NumPy computes SVD and the inverse matrix in double precision.